lambdas/testFunction/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst

```diff
@@ -0,0 +1 @@
+/Users/cygnus/Documents/cs428/chartr/lambdas/testFunction/src/main/java/chartr/TestFunction.java
```

Comment:
Should we be Git ignoring this?

Reply:
It's still hooked up to an endpoint on Chartr and serves as a good reference for how to write the lambda code (in our environment) in Java. I would prefer leaving it in the codebase for now.

Reply:
OK! Everything else looks good to me then